src/config.json

```diff
@@ -0,0 +1,4 @@
+{
+	"clientKey": "FKx1oF94M0OuoDW0YDyAx6tlelUvR3wm", 
+	"clientSecret": "CdaU65M2wJ7G_HJo3eL_NNA-3IOuAbx0llsW46gGoE7FzTIwnqpGsM57tM2AfoMQ"
+}
```

Comment:
nit: add an empty line at the  end of the file.

Reply:
Thanks, Done